src/index.html

```diff
@@ -11,7 +11,178 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+   <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form__list">
+        <legend>Personal information</legend>
+        <div class="form-box">
+          <label for="sname" class="form__list-item">Surname:
+            <input
+              type="text"
+              id="sname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-box">
+          <label for="name"  class="form__list-item">Name:
+            <input
+              type="text"
+              id="name"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-box">
+          <label for="old" class="form__list-item">How old are You?
```

Comment:
```suggestion
          <label for="sname" class="form__list-item">
            Surname:
            <input
              type="text"
              id="sname"
              autocomplete="off"
            >
          </label>
        </div>

        <div class="form-box">
          <label for="name"  class="form__list-item">
            Name:
            <input
              type="text"
              id="name"
              autocomplete="off"
            >
          </label>
        </div>

        <div class="form-box">
          <label for="old" class="form__list-item">
          How old are You?
```

move text to a new line

Reply:
check all cases